charts/openshift-metering/templates/presto/hive-metastore-statefulset.yaml

```diff
@@ -120,6 +120,14 @@ spec:
 {{- end }}
         resources:
 {{ toYaml .Values.presto.spec.hive.metastore.resources | indent 10 }}
+      - name: ghostunnel-server
+        command: ["ghostunnel"]
+        args: ["server", "--listen", "localhost:8443", "--target", "localhost:9083", "--key", "/opt/hive/tls/tls.key", "--cert", "/opt/hive/tls/tls.crt", "--cacert", "/opt/hive/tls/ca.crt", "--allow-cn", "Any"]
+        image: quay.io/openshift/origin-ghostunnel
```

Comment:
Your container definition exposes no `ports`. It should at least expose 8443. Also, the hive-metastore service could probably be updated to include the new port so we can connect to ghostunnel.